examples/transport-rr/generate_consistent.py

```diff
@@ -39,7 +39,7 @@
 pMax = 1    # maximum momentum in units of m_e*c
 Np   = 150  # number of momentum grid points
 Nxi  = 6    # number of pitch grid points
-tMax = 2e-3 # simulation time in seconds
+tMax = 1e-3 # simulation time in seconds
```

Comment:
this example fails with tMax=2e-3 because the nonlinear solver does not converge. I suspect that the reasion is the missing off-diagonal jacobians in the external boundary condition, but I'm not entirely sure. You end up with weak steady oscillations in the currents and E-field that do not decrease in amplitude with iterations.